### PR TITLE
Updates for AWS deployment

### PR DESCRIPTION
**Pull request by bmallred**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -16,10 +18,14 @@ all: clean setup lint test build
 clear:
 	@rm -rf ./errors
 
+reset-permissions: clear
+	$(info Resetting permissions)
+	@docker-compose run --rm deps ./bin/permissions $(uid) $(gid) 2>errors
```
Comment (bmallred) [inline Makefile]: Permissions are now applied by the UID and GID of the account executing the command.

File: Makefile [hunk 2]
```diff
@@ -145,9 +151,46 @@ deploy-react:
...
 #
-react: test-react build-react
-go: test-go build-go
+react: test-react build-react reset-permissions
+go: test-go build-go reset-permissions
 
+#
+# seccomp
+#
+seccomp:
```
Comment (bmallred) [inline Makefile]: Steps to perform to create a new `seccomp` template.

File: api/bin/seccomp [hunk 3]
```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+strace -ff -o /tmp/strace/eapp.strace $@;
```
Comment (bmallred) [inline api/bin/seccomp]: Run a command with `strace` turned on.

File: api/bin/seccomp-collate [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+						"name": "eApp API",
+						"names": [
+EOF
+
+# Add sorted syscalls to the JSON
+cat /tmp/strace/eapp.strace* \
+		| awk '/^[a-zA-Z0-9_]+\(/' \
+		| awk -F "(" '{print "\t" "\t" "\t" "\t" "\t" "\t" "\t" "\x22" $1 "\x22" ","}' \
+		| sort -u \
+		>> eapp-seccomp.json;
```
Comment (bmallred) [inline api/bin/seccomp-collate]: Collate all trace files extracting only the system calls and outputting the unique calls in a sorted fashion with the JSON structure.

File: Makefile (hunk 2 above)
Comment (floresj) [reply Makefile]: The seccomp stuff was really cool :+1:

**Pull request merged**